cmd/minikube/cmd/service.go

```diff
@@ -148,6 +148,10 @@ You may select another namespace by using 'minikube service {{.service}} -n <nam
 					out.String(fmt.Sprintf("%s\n", serviceURLs))
 				}
 			}
+			// check whether all pods of this service is crashed
+			if err := service.CheckServicePods(cname, svc.Name, namespace); err != nil {
+				exit.Error(reason.SvcUnreachable, "error checking service", err)
```

Comment:
This is a little misleading, if we successfully check the pods but they're not running we still return an error, which will output `error checking service`. But we successfully checked the service, it's just that the pods are not running. Something like `service not available` might be better as it will encapsulate the true meaning